components/resources/ResourceCard.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { TouchableOpacity, Linking } from 'react-native';
+import {
+  Card,
+  ContentContainer,
+  ContentText,
+  LinkSubhead,
+  IconContainer
+} from '../../styled/resources';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Colors from '../../assets/Colors';
+
+class ResourceCard extends React.Component {
+  render() {
+    return (
+      <TouchableOpacity
+        onPress={() => Linking.openURL(this.props.resourceCard.url)}>
+        <Card>
+          <ContentContainer>
+            <LinkSubhead>{this.props.resourceCard.title}</LinkSubhead>
+            <ContentText>{this.props.resourceCard.description}</ContentText>
+            <IconContainer>
```

Comment:
First of all, ContentContainer seems redundant. In general, if you find yourself using a container directly inside a container, you should probably consider if you should restructure or somehow consolidate this styling.

HOWEVER, I think you should restructure this card to nest like this: 
- Card
   - ContentContainer
      - ...text components...
   - FontAwesome5

 You wouldn't need IconContainer at all if you move the `<FontAwesome5>...` out of the `ContentContainer`.

With this, your `ContentContainer` can use `flex-direction: column` dedicated to the vertical alignment of the title and description, and `Card` can use `flex-direction: row` for the horizontal spacing of text content and the link icon. This way you can significantly simplify the repeated centering of several containers, and you can get consistent spacing and placement of the icon, rather than letting the text run through/past the icon like in your screenshot. 
![image](https://user-images.githubusercontent.com/21160510/76155964-61372e80-60a8-11ea-9607-35caf31df006.png)